alerter.c

```diff
@@ -1,8 +1,15 @@
 #include <stdio.h>
 #include <assert.h>
 
+
+#define TEST_ENV    (1)
+#define PROD_ENV    (2)
+
+#define ENV     TEST_ENV
+
 int alertFailureCount = 0;
 
+//for testing purpose
 int networkAlertStub(float celcius) {

```

Comment:
Stub is part of production code in the same file. This will get compiled and get shipped to field along with production code. Hence it is necessary to keep test code and production code separate in order to exclude test code for deployment.